src/main/java/use_case/LeaveReview/LeaveReviewInteractor.java

```diff
@@ -15,13 +13,13 @@ public LeaveReviewInteractor(LeaveReviewDoctorDataAccessInterface doctorDAO,
     }
     @Override
     public void execute (LeaveReviewInputData leaveReviewInputData) {
-      //  if (doctor not in patient.orderhistory)
-        //userPresenter.prepareFailView("You can only rate Doctors you have received service from")
-        // else:
 
-    //    review.doctor.settotalreviews(review.doctor.gettotalreviews() + 1);
-     //   review.doctor.
+        Doctor doctor = doctorDAO.getDoctor(leaveReviewInputData.getDoctor());
+        doctor.setTotalReviews(doctor.getTotalReviews() + 1);
+        int temp = doctor.getTotalReviews();
```

Comment:
Here are few suggestions:
- Variable name could be clearer and more descriptive